tests/contrib/flask/__init__.py

```diff
@@ -7,13 +8,33 @@
 from tests.utils import TracerTestCase
 
 
+class DDFlaskTestClient(FlaskClient):
```

Comment:
The following tests fail without this change to the FlaskcLient: https://app.circleci.com/pipelines/github/DataDog/dd-trace-py/21330/workflows/ad536a91-4d51-4cd8-b4bf-947207c5c5ae/jobs/1451641